src/actions/__tests__/onboardingActions.test.js

```diff
@@ -303,9 +303,8 @@ describe('Onboarding actions', () => {
       { payload: false, type: SET_FETCHING_TOTAL_BALANCES },
 
       { type: SET_FETCHING_HISTORY, payload: true },
-      { type: SET_HISTORY, payload: { [mockArchanovaAccount.id]: { ethereum: [] } } },
-      { type: SET_FETCHING_HISTORY, payload: false },
       { type: SET_FETCHING_RATES, payload: true },
+      { type: SET_HISTORY, payload: { [mockArchanovaAccount.id]: { ethereum: [] } } },
```

Comment:
Why is there no matching `SET_FETCHING_HISTORY false`?

Reply:
Because there is a promise inside an action and action is not waited therefore race condition happens on test itself and it's never updated. We have similar cases cross all tests.

Reply:
Ok, just wanted to bring your attention to it.

Reply:
I'm aware about this, thanks.